handsome-hedgehogs/workforce-management
# may need some css, but functional

## Status
**Ready**

## Description 
User can see a list of departments when they click on the nav bar department button
User can create a new department
User can see a list of employees when viewing department details

## Steps to Test or Reproduce
1. git fetch
2. git checkout js-departments
3. run solution
4. confirm *site/departments displays the list of departments
7. click on a department to go to details
4. confirm details displays a list of employees in that department
9. on department screen, confirm a "create new" button works
34. attempt to create a new department by entering a name

## Impacted Areas in Application
departmentscontroller.cs
views/departments

## Tickets 
#5 
#6 
#7
